dcrjson/error_test.go

```diff
@@ -58,11 +51,11 @@ func TestError(t *testing.T) {
 		want string
 	}{
 		{
-			Error{Message: "some error"},
+			Error{Description: "some error"},
```

Comment:
Mind making these consistent with the newer test style for the braces as done below?